scripts/migrate-07_0.7.1-to-0.8.0.js

```diff
@@ -126,7 +126,7 @@ function runDatabaseMigration() {
             $push: {
                 'config.queryBuilder': {
                     name : 'rocketchatsearch',
-                    displayName : 'rocketchatsearch',
+                    displayName : 'Ähnliche Nachrichten',
```

Comment:
This has nothing to do with this pull request and should be reverted

Reply:
I reverted this change. Thanks for the advice.